### PR TITLE
Add speaker profiles

### PR DESCRIPTION
**Pull request by amryarios**
Add speaker profiles :

Soares Chen : http://soareschen.com

Bernard Leong : http://www.bernardleong.com/
                       https://twitter.com/bleongcw
                       https://www.facebook.com/bernardleong
                       https://github.com/bleongcw

Rizky Ariestiyansyah : https://www.linkedin.com/in/ariestiyansyah/
                               https://twitter.com/ariestiyansyah
                               https://github.com/ariestiyansyah
                               http://oonlab.com/

### CODE REVIEW

**Review by akshatnitd — COMMENTED** [review]
The changes looks good. If you could provide the deployment link, I can approve and finally merge it :)

**Comment by amryarios** [comment]
Meaning? I still do not understand the meaning of providing link spread? please help how?

**Comment by akshatnitd** [comment]
You need to deploy the forked version of your website. Check https://pages.github.com for help.

**Review by 0x48piraj — COMMENTED** [review]
{your_username}.github.io/{your_repo}

**Comment by akshatnitd** [comment]
Yes, it would be https://amryarios.github.io/2016.fossasia.org in your case, but you also need to update the latest changes in the `gh-pages` branch of your repo.

**Comment by amryarios** [comment]
Okay now I understand :) wait I'll do it

**Comment by amryarios** [comment]
capture
Has it worked?

**Review by 0x48piraj — APPROVED** [review]
LGTM ! :rocket:

**Review by lunaticmonk — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -752,7 +752,7 @@ <h1>Speakers</h1>
 											NodeSchool is an open source project run by volunteers with two goals: to create high quality programming curriculum and to host community learning events.
 										</div>
 										<div class="social-links vertical-align">
-											<a href="#"><i class="icon fa fa-external-link"></i></a>
+											<a href="https://soareschen.com"><i class="icon fa fa-external-link"></i></a>
 											<a href="https://github.com/soareschen"><i class="icon fa fa-github"></i></a>
```
Comment (akshatnitd) [inline index.html]: This link seems to be dead. Change `https` to `http` and it ll work.

**Review by akshatnitd — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (amryarios) [reply index.html]: Oke, Wait...

**Comment by amryarios** [comment]
Please Review Again

**Comment by akshatnitd** [comment]
Please squash the commits and it will be merge ready. :)

For help related to squashing the commits, check this out, https://gist.github.com/patik/b8a9dc5cd356f9f6f980

**Review by ParthS007 — CHANGES_REQUESTED** [review]
Please resolve conflicts 👍

**Comment by amryarios** [comment]
@ParthS007 @akshatnitd how is it right? or there is still a problem

**Comment by arjunpandey2003** [comment]
Try to squash your commits!

**Comment by arjunpandey2003** [comment]
For help related to squashing the commits, check this out, https://gist.github.com/patik/b8a9dc5cd356f9f6f980

File: index.html [hunk 2]
```diff
@@ -781,6 +781,8 @@ <h1>Speakers</h1>
 										<div class="social-links vertical-align">
 											<a href="https://swarajyamag.com/author/17549"><i class="icon fa fa-external-link"></i></a>
 											<a href="#"><i class="icon fa fa-github"></i></a>
+											<a href="#"><i class="icon social_twitter"></i></a>
```
Comment (abishekvashok) [inline index.html:784]: Is this really wanted as the twitter link is also present.

File: index.html [hunk 3]
```diff
@@ -821,6 +823,10 @@ <h1>Speakers</h1>
 											It's great to be at FOSSASIA in Singapore! Fantastic atmosphere with lots of friendly devs, designers and business folks.
 										</div>
 										<div class="social-links vertical-align">
+											<a href="http://www.bernardleong.com/"><i class="icon fa fa-external-link"></i></a>
+											<a href="https://github.com/bleongcw"><i class="icon fa fa-github"></i></a>
```
Comment (abishekvashok) [inline index.html:827]: Twitter link already present

**Review by abishekvashok — CHANGES_REQUESTED** [review]

**Review by vaibhavsingh97 — CHANGES_REQUESTED** [review]
Please squash your commits.

**Comment by amryarios** [comment]
https://amryarios.github.io/2016.fossasia.org Please check again ?

**Review by ParthS007 — COMMENTED** [review]
@amryarios  Please resolve conflicts else looks fine 👍

**Comment by amryarios** [comment]
has it worked?

**Comment by arjunpandey2003** [comment]
looks good

**Comment by arjunpandey2003** [comment]
Just in thePR description write the number of the issue fixed

**Review by arjunpandey2003 — APPROVED** [review]

**Review by nzec — APPROVED** [review]
Looks k.

**Comment by amryarios** [comment]
untitleda Finish please check again
https://amryarios.github.io/2016.fossasia.org

**Review by supunTE — APPROVED** [review]
LGTM! can you center align photos.

**Review by harshit98 — APPROVED** [review]

**Pull request merged**